### DISCUSSION TITLE
When I run the client why does the stuff I change not change.

### DISCUSSION THREAD

**Discussion by bruh2899** [MCPHackers/RetroMCP-Java#116] [Q&A]
When I run the client it is still just vanilla minecraft instead of modded minecraft

**Comment by mudkipdev** [comment]
Need more info

**Reply by bruh2899** [reply]
When I go to RetroMCP-J window, press the Launch Client button the drops I changed for mobs are still the vanilla drops

**Reply by PhoenixVX** [reply]
You need to recompile first, then launch client.

**Reply by bruh2899** [reply]
I did that and nothing changed

**Reply by mudkipdev** [reply]
You probably opened the workspace or edited the source code wrong
